model/Manager.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class Manager
+{
+    protected function databaseConnect()
```

Comment:
Cette méthode peut aussi être mise dans un constructeur.
$database pourrait être un attribut protected de la classe.
$this->database pourrait donc être utilisé dans la classe enfant et ca te permettrait d'avoir qu'une seul connexion à ta db